### ISSUE TITLE
archive/tar: sockets not supported

### ISSUE THREAD

**Issue by koos147** [muesli/docker-backup#9]
I am runing docker on debian
`Linux version 4.19.0-12-amd64 (debian-kernel@lists.debian.org) (gcc version 8.3.0 (Debian 8.3.0-6)) #1 SMP Debian 4.19.152-1 (2020-10-18)
`
when i run `docker-backup --all --tar` i receive an error `archive/tar: sockets not supported`
when i do the same without the --tar i get 2 files for every docker.
```
portainer_portainer-ce-6bc5f0099ce2b8b9ecce0a34510585901661376e7341eeea2673f0fbccb77d68.backup.files
portainer_portainer-ce-6bc5f0099ce2b8b9ecce0a34510585901661376e7341eeea2673f0fbccb77d68.backup.json
```
in the .files file is only one line of text pointing to the .json file.
the json shows the config of the image.
however i was expecting a backup of the image itself.

what is going wrong?

**Comment by muesli** [comment]
This error may have been fixed today.

The behavior is normal btw: without the `--tar` flag, `docker-backup` only creates a JSON file containing the container's metadata as well as a list of files, which you can feed to an external backup program.

**Comment by GoldraK** [comment]
I have the same problem with tar -> archive/tar: sockets not supported I clone today the repo but not fix this problema.

Thanks for your time.

**Comment by muesli** [comment]
Thanks for your feedback, @GoldraK! Fixed by #15.
